packages/auth/src/Auth.ts

```diff
@@ -379,6 +381,21 @@ export class AuthClass {
 							data,
 							`${username} has signed up successfully`
 						);
+						if (autoSignIn) {
+							Hub.listen('auth', resp => {
+								let { payload } = resp;
```

Comment:
Nit: you could destructure inline like the `AuthClass` does [here](https://github.com/aws-amplify/amplify-js/blob/5e8264963c19b50069e9acc2b692d8e6a1a0a201/packages/auth/src/Auth.ts#L122).
```suggestion
							Hub.listen('auth', ({ payload }) => {
```